rerun.go

```diff
@@ -212,11 +221,18 @@ func rerun(buildpath string, args []string) (err error) {
 		return
 	}
 
+	// construct regexp obj, unused on error (assuming Compile returns nil if so).
+	var ignoreRe *regexp.Regexp
+	ignoreRe, err = regexp.Compile(*ignore_re_str)
+	if err != nil {
+		log.Print(err)
+	}
 	for {
 		// read event from the watcher
 		we, _ := <-watcher.Event
 		// other files in the directory don't count - we watch the whole thing in case new .go files appear.
-		if filepath.Ext(we.Name) != ".go" {
+		if fn := filepath.Base(we.Name) ;
+		(ignoreRe != nil && ignoreRe.FindString(fn) != "") || filepath.Ext(fn) != ".go" {
```

Comment:
please run gofmt - this line doesn't look like what gofmt would come up with.

Reply:
OK, done in later commit.